sqlmesh/core/model/definition.py

```diff
@@ -1784,6 +1785,84 @@ class AuditResult(PydanticModel):
     blocking: bool = True
 
 
+def create_model_blueprints(
+    gateway: t.Optional[str | exp.Expression],
+    blueprints: t.Any,
+    get_variables: t.Callable[[t.Optional[str]], t.Dict[str, str]],
+    loader: t.Callable[..., Model],
+    **loader_kwargs: t.Any,
+) -> t.List[Model]:
+    path = loader_kwargs.get("path")
+    dialect = loader_kwargs.get("dialect")
+
+    if not blueprints:
```

Comment:
This entire block can be a separate function dedicated to resolving the blueprint value

Reply:
Yep, the body just seemed small/simple enough to me, so I avoided creating more helpers for this. Happy either way, let me know what's your preference

Reply:
I think this will make the method simpler to comprehend. The purpose of this method is to get the list of blueprints and iterate over them. Instead it:
1. Parses blueprints
2. Extracts blueprint variables
3. Parses gateway
4. Iterates over blueprints and loads models.

I think it should only be doing (4) with everything else delegated to subroutines.

Reply:
Got it, sounds good.

Reply:
Addressed. Refactored both this section and the blueprint variable extraction out into helpers.